source/agora/node/FullNode.d

```diff
@@ -401,6 +374,34 @@ public class FullNode : API
 
     public void start ()
     {
+        // If we have handlers defined in the config
```

Comment:
Why the move ?

Reply:
We have to create Flash nodes after Agora nodes are created. So at the time FullNode ctor runs, there are no Flash nodes registered, even if they are put in the config as block handlers. So defering the lookup to `start()` makes sure that all nodes (Agora & Flash) are instantiated and registered.